docker-compose.yml

```diff
@@ -6,11 +6,13 @@ services:
     ports:
       - 25565:25565
     environment:
+      TANGIA_ENV: "STAGING"
```

Comment:
might not be a good idea to put into main if we are asking them to copy this

Reply:
good point